ftag/flavours.yaml

```
@@ -140,6 +140,43 @@
   colour: "#e76f51"
   category: lepton-decay
 
+# PGID labelling: taken from https://pdg.lbl.gov/2023/reviews/rpp2023-rev-monte-carlo-numbering.pdf
+- name: D0meson
+  label: $D^0$ in $c$-hadron decay
```

```suggestion
  label: $D^0$ in $c$-jet
```
and similar for other labels